packages/web3-types/src/eth_contract_types.ts

```diff
@@ -168,8 +168,4 @@ export interface ContractOptions {
 	 * The max fee per gas to use for transactions.
 	 */
 	maxFeePerGas?: Uint;
-	/**
-	 * Ignore gas price, turn on for metamask suggestion fee
-	 */
-	ignoreGasPricing?: boolean;
```

Comment:
in case some one is using this will be breaking change

Reply:
this was from yesterdays merged pr, [packages/web3-types/src/eth_contract_types.ts ](https://github.com/web3/web3.js/commit/69187c5ed45fa771d46d731277b0292c15c5546a#diff-7d705ade2c517fe375015c6043e70ed20451e8e81f1cb9e7fe85d7cbc5a23b3dR169) included by accident